### ISSUE TITLE
Migrate from Nominatim to Google Maps

### ISSUE THREAD

**Issue by taymonbeal** [rtcharity/hubreboot#264]
Nominatim is very much not intended for production traffic, and has very strict rate limits that we're quite likely to run into. They blocked my IP while I was running the Airtable migration script on my laptop.

**Comment by michaltrzesimiech** [comment]
Isn't this done by now?

**Comment by taymonbeal** [comment]
No, we're using Google Maps for the actual map but not for geocoding.

**Comment by patbl** [comment]
I'll start on this if no one else has. It seems like it'll be pretty straightforward.

**Comment by vagarwalla** [comment]
Check related merge request.

**Comment by vagarwalla** [comment]
It's not free anymore, blocked by #1041
